actions/class.ItemImport.php

```diff
@@ -33,6 +37,8 @@
  */
 class taoItems_actions_ItemImport extends tao_actions_Import
 {
+    public const FEATURE_FLAG_TABULAR_IMPORT = 'FEATURE_FLAG_TABULAR_IMPORT';
```

Comment:
As we also can have feature flag do disable, maybe a suffix to be 100% clear. Just a suggestion. 

```suggestion
    public const FEATURE_FLAG_TABULAR_IMPORT = 'FEATURE_FLAG_TABULAR_IMPORT_ENABLED';
```